core/src/main/java/org/wso2/testgrid/core/command/GenerateTestPlanCommand.java

```diff
@@ -213,33 +215,21 @@ private void setUniqueScriptName(List<Script> scripts) {
      */
     private String createTestPlanGenDirectory(Product product) throws CommandExecutionException {
         try {
-            String directoryName = product.getId();
+            String directoryName = product.getName();
             String testGridHome = TestGridUtil.getTestGridHomePath();
-            Path directory = Paths.get(testGridHome, directoryName).toAbsolutePath();
+            Path directory = Paths.get(testGridHome, directoryName, PRODUCT_TEST_PLANS_DIR).toAbsolutePath();
 
             // if the directory exists, remove it
             removeDirectories(directory);
 
-            logger.info(StringUtil.concatStrings("Creating test directory : ", directory.toString()));
-            Path createdDirectory = createDirectories(directory);
-            logger.info(StringUtil.concatStrings("Directory created : ", createdDirectory.toAbsolutePath().toString()));
+            Path createdDirectory = Files.createDirectories(directory.toAbsolutePath());
```

Comment:
the variable 'directory' is already has the absolute path as per the line 220. So I think we can just use,
`Path createdDirectory = Files.createDirectories(directory);
`
WDYT?